app/API/V1/Entities/User.php

```diff
@@ -40,46 +44,119 @@
  */
 class User extends EntityAbstract implements HasRolesContract, HasPermissionsContract, ExtractableContract, AuthenticatableContract, NotifiableEntityContract
 {
-    use Authenticatable, Blameable, SoftDeleteable, IpTraceable, Timestampable, HasPermissionsOptimizedTrait, ExtractorOptionsTrait, NotifiableTrait;
+    use Authenticatable, Blameable, SoftDeleteable, IpTraceable, Timestampable, HasPermissionsOptimizedTrait, ExtractorOptionsTrait, NotifiableTrait, GenerateRandomString;
 
     /**
      * @ORM\Id
-     * @ORM\GeneratedValue(strategy="AUTO")
-     * @ORM\Column(type="integer", name="id")
-     * @var int $id
+     * @ORM\Column(type="string", name="id")
+     * @ORM\GeneratedValue(strategy="CUSTOM")
+     * @ORM\CustomIdGenerator(class="\TempestTools\Common\Doctrine\Generator\SecureUniqueIdGenerator")
+     * @var string $id
      */
     public $id;
 
     /**
-     * @ORM\Column(type="string", nullable=true, name="name")
-     * @var string $name
+     * @ORM\Column(type="string", name="identification_key")
+     * @ORM\GeneratedValue(strategy="CUSTOM")
+     * @ORM\CustomIdGenerator(class="\TempestTools\Common\Doctrine\Generator\SecureUniqueIdGenerator")
+     * @var string $identificationKey
+     */
+    public $identificationKey;
+
+    /**
+     * @ORM\Column(type="string", name="instadat_uuid", nullable=true)
      * @Gedmo\Versioned
+     * @var string $instadatUUid
      */
-    protected $name;
+    protected $instadatUUid;
```

Comment:
@jerazo Can we change this in open source and on the client branch to $thirdPartyUuid? That way they can stay in sync, with out being specific to either project.

Reply:
fixed